content/docs/pipelines/sdk/output-viewer.md

```diff
@@ -400,12 +432,19 @@ management tools.
 
 - `source`
 
+**Optional metadata fields:**
+
+- `storage`
+
 The `web-app` viewer provides flexibility for rendering custom output. You can
 specify an HTML file that your component creates, and the Kubeflow Pipelines UI
 renders that HTML in the output page. The HTML file must be self-contained, with
 no references to other files in the filesystem. The HTML file can contain
 absolute references to files on the web. Content running inside the web app is
-isolated in an iframe and cannot communicate with the Kubeflow Pipelines UI.
+sandboxed in an iframe and cannot communicate with the Kubeflow Pipelines UI.
+
+You can optionally specify `'storage': 'inline'` to embed raw html in `source`
```

Comment:
Suggestion(shorten)

Specify `'storage': 'inline'` to store raw html in the `source` field.